langing-page/main.js

```diff
@@ -0,0 +1,10 @@
+function changeTheme() {
+  var btn = document.querySelector(".bg");
+  if (btn.classList.contains("dark")) {
+    btn.classList.remove("dark");
```

Comment:
Use
```
btn.classList.toggle('dark')
```
and remove if conditions. Toggle would add and remove when the function is called.